<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cybertetris</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="glass">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>300</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pause">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tetris</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>221</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="ScoreLabel">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Score</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Score">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LinesLabel">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rows</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lines">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LevelLabel">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Class</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Level">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TimerIntervalLabel">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TimerInterval">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="nextShapeGlass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>92</width>
      <height>92</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>92</width>
      <height>92</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nextShape">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimerShowLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TimerShow">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>MV Boli</family>
        </font>
       </property>
       <property name="text">
        <string>Hi!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>MV Boli</family>
        </font>
       </property>
       <property name="text">
        <string>Gold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="coins">
       <property name="font">
        <font>
         <family>Tetris</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="name">
       <property name="font">
        <font>
         <family>MV Boli</family>
        </font>
       </property>
       <property name="text">
        <string>tourist</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="item1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Cybertetris-Desktop_Qt_5_12_MinGW_64_bit-Debug/resources/items/1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="item2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Cybertetris-Desktop_Qt_5_12_MinGW_64_bit-Debug/resources/items/2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="item3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Cybertetris-Desktop_Qt_5_12_MinGW_64_bit-Debug/resources/items/3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="enemyGlass">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>300</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tetris</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pause_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tetris</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="music">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Cybertetris-Desktop_Qt_5_12_MinGW_64_bit-Debug/resources/setting/music1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="saveGame"/>
    <addaction name="loadGame"/>
   </widget>
   <widget class="QMenu" name="set">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="storeMenu"/>
    <addaction name="setting"/>
    <addaction name="backMenu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="set"/>
  </widget>
  <action name="saveGame">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="loadGame">
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="setting">
   <property name="text">
    <string>游戏设置</string>
   </property>
  </action>
  <action name="backMenu">
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </action>
  <action name="storeMenu">
   <property name="text">
    <string>商城</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
